R/convert.R

```diff
@@ -64,7 +64,12 @@ convert <- function(from, to, ...) {
     }
     from
   } else {
-    method_lookup_error("convert", c("from", "to"), dispatch)
+    msg <- paste0(
+      "Can't find method for generic `convert()` with dispatch classes:\n",
+      "- from: ", obj_desc(from), "\n",
+      "- to  : ", class_desc(to), "\n"
+    )
+    stop(msg, call. = FALSE)
```

Comment:
Out of curiosity, why not use `method_lookup_error()`? The only thing that looks like it changed in the snapshot is the usage of `<` `>`

Reply:
Because `to` is a class, not an object.